backend/apps/ai/management/commands/ai_create_chapter_chunks.py

```diff
@@ -0,0 +1,216 @@
+"""A command to create chunks of OWASP chapter data for RAG."""
+
+import os
+import time
+from datetime import UTC, datetime, timedelta
+
+import openai
+from django.core.management.base import BaseCommand
+
+from apps.ai.common.constants import (
+    DEFAULT_LAST_REQUEST_OFFSET_SECONDS,
+    MIN_REQUEST_INTERVAL_SECONDS,
+)
+from apps.ai.models.chunk import Chunk
+from apps.owasp.models.chapter import Chapter
+
+
+class Command(BaseCommand):
+    help = "Create chunks for OWASP chapter data"
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "--chapter-key", type=str, help="Process only the chapter with this key"
+        )
+        parser.add_argument(
+            "--active-only", action="store_true", help="Process only active chapters"
```

Comment:
Active only should be the default. Optionally we may want to process inactive too.